nvc/nvc.go

```diff
@@ -0,0 +1,147 @@
+// Package nvc implements a parser for nvc archive files.
+//
+// The format of an archive is described below. All header values are stored on disk as little endian.
+//
+//   1. NVC magic bytes (8 bytes)
+//   2. Number of entries in the archive's table of contents (32-bit little endian unsigned integer)
+//   3. Table of Contents entries (192 * n bytes, where n is the previously-specified number of entries)
+package nvc
+
+import (
+	"compress/zlib"
+	"encoding/binary"
+	"errors"
+	"fmt"
+	"hash/fnv"
+	"io"
+)
+
+// EntryFlags describes how a file is stored on disk
+type EntryFlags uint32
+
+const (
+	Uncompressed EntryFlags = 0
+	Compressed   EntryFlags = 1
+	Encrypted    EntryFlags = 3
+
+	magic = "nvc1d\x00\x00\x00"
+)
```

Comment:
should we document these constants?

Reply:
Yeah, best practice is to document everything that's exported. Done in 8f449c11878837cbb813afc3a0d5b12ee0814b89

Also I renamed the flag constants to be more idiomatic